x-pack/test/case_api_integration/common/lib/utils.ts

```diff
@@ -622,19 +699,6 @@ export const createComment = async ({
   return theCase;
 };
 
-export const getAllUserAction = async (
```

Comment:
Removing this one in favor of `getCaseUserActions` because it takes the `auth` field.